ewms_pilot/pilot.py

```diff
@@ -44,10 +44,10 @@ class FileType(enum.Enum):
 def get_last_line(fpath: Path) -> str:
     """Get the last line of the file."""
     with fpath.open() as f:
-        try:
-            return f.readlines()[-1].rstrip()
-        except IndexError:
-            return ""
+        line = ""
+        for line in f.readlines():
```

Comment:
```suggestion
        for line in f:
```
I think this is the right way to read a file line by line?